MediaPlayer/MediaControls.js

```diff
@@ -566,6 +585,7 @@ const MediaControlsDecorator = hoc((config, Wrapped) => {
 			this.actionGuideHeight = 0;
 			this.animation = null;
 			this.bottomComponentsHeight = 0;
+			this.moreComponentsSpotlightId = 'moreComponents';
```

Comment:
Where does the code come from?

there is default prop already.

Reply:
Moved from the https://github.com/enactjs/sandstone/pull/807.